code/src/index.css

```diff
@@ -1,13 +1,273 @@
+
+.main-container h1{
+  color: white;
+  margin: 20px;
+}
+
+.main-container{
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  margin: 10px;
+  position: absolute;
+}
+
+.main-container button{
+  border-radius: 20px;
+  width: 100px;
+  height: 40px;
+  background-color: #fd9a89;
+  font-weight: bold;
+  color: white;
+  border: 0 none;
+  cursor: pointer;
+  padding: 10px 5px;
+  margin: 10px 5px;
+}
+
+.main-container button:hover{
+  background-color: transparent;
+    border: 2px solid #fd9a89;
+    cursor: pointer;
+}
+
+.App {
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+body {
+  margin: 0%;
+  padding: 0%;
+  font-family: montserrat, arial, verdana;
+}
+
+.form-container .header .x{
+  position: absolute;
+  width: 15px;
+  height: 3px; 
+  top: 20px;
+  right: 5px;
+  transform:translate(-50%, -50%);
+  background: transparent;
+  cursor: pointer;
+}
+
+.x::before , .x::after{
+  content: '';
+  position: absolute;
+  width: inherit;
+  height: inherit;
+  background: white;
+}
+
+.x::before{
+  transform:rotate(50deg);
+}
+
+.x::after{
+  transform:rotate(130deg);
+}
+
+.form {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.form-container {
+  width: 400px;
+  height: 400px;
+  background-color: black;
+  border-radius: 8px;
+  box-shadow: 0 0 15px 1px rgba(0, 0, 0, 0.4);
+  display: flex;
+  position: relative;
+  flex-direction: column;
+}
+
+.form-container h1{
+  margin-top: 40px;
+  color: white;
+}
+
+.form-container .header {
+  flex: 20%;
+  display: flex;
+  justify-content: center;
+}
+
+.form-container .body {
+  flex: 60%;
+}
+
+.form-container .footer {
+  flex: 20%;
+  display: flex;
+  justify-content: center;
+}
+
+.form-container .footer button {
+  border-radius: 20px;
+  width: 100px;
+  height: 40px;
+  background-color: #fd9a89;
+  font-weight: bold;
+  color: white;
+  border: 0 none;
+  cursor: pointer;
+  padding: 10px 5px;
+  margin: 20px 5px;
+}
+
+.form-container button:hover{
+  background-color: transparent;
+    border: 2px solid #fd9a89;
+    cursor: pointer;
+}
+
+.sign-up-container {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.sign-up-container input {
+  margin: 5px;
+  width: 200px;
+  height: 40px;
+  padding-left: 5px;
+  font-size: 20px;
+}
+
+.personal-info-container {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-size: 20px;
+}
+
+.personal-info-container input {
+  margin: 5px;
+  width: auto;
+  height: auto;
+  font-size: 50px;
+  cursor: pointer;
+}
+
+input {
+  border: 2px solid;
+  border-radius: 5px;
+  background-color: rgb(185, 185, 185);
+  vertical-align: middle;
+}
+
+input[name="destination"] {
```

Comment:
wow! what does this do?

Reply:
This allows you to change the styling of the radio button, not sure if this one is active since I have another one below. Was a bit stressed out handing this in:D